setup.py

```diff
@@ -21,11 +21,12 @@
     packages=find_packages(),
     include_package_data=True,
     install_requires=[
-        "torch==1.7.1",
-        "pytorch-lightning==1.0.2",
-        "faiss-gpu==1.6.3",
+        # "torch>=1.7.1",
+        # "pytorch-lightning>=1.0.2",
```

Comment:
Given the rapid development of the package, better to add a version requirement.